ReadMe.md

```diff
@@ -18,11 +18,13 @@ Backend Setup (Flask)
 First, create a virtual environment to isolate the project dependencies. Use the following commands in your terminal or PowerShell:
 
  Create a virtual environment
-py -m venv "E:\Research\Vue-Flusk\dnr\venv"
+ py -m venv "E:\Research\Vue-Flusk\dnr\venv" (for windows)
+ python3 -m venv myenv (for Mac)
 
- Activate the virtual environment (for Windows PowerShell)
-.\venv\Scripts\Activate.ps1
+ Activate the virtual environment
+.\venv\Scripts\Activate.ps1 (for Windows PowerShell)
```

Comment:
nit: grouping these types of instructions by platform instead of by "action" is better for the read.

ex:

### Init Env
#### Mac 
1. python3 -m venv myenv
2. source env/bin/activate 

####  Windows
1. etc
2. etc